nix/pkgs/frontend.nix

```diff
@@ -26,9 +26,12 @@ pkgs.buildNpmPackage (
         lib.attrsets.mapAttrs (
           name: pkgInfo:
           let
+            resolvedParts = lib.splitString "#" pkgInfo.resolved;
```

Comment:
Maybe I should have done this in the initial implementation :see_no_evil: , but maybe add a comment why this is done / how the `pkgInfo.resolved` link looks like?

Reply:
Refactored it a bit :)

Reply:
Nice